patches/tModLoader/Terraria/GameContent/UI/EmoteBubble.cs.patch

```diff
@@ -1,19 +1,42 @@
 --- src/TerrariaNetCore/Terraria/GameContent/UI/EmoteBubble.cs
 +++ src/tModLoader/Terraria/GameContent/UI/EmoteBubble.cs
-@@ -1,3 +_,4 @@
+@@ -1,5 +_,7 @@
 +using Terraria.ModLoader;
  using Microsoft.Xna.Framework;
  using Microsoft.Xna.Framework.Graphics;
++using ReLogic.Content;
  using System;
-@@ -9,7 +_,7 @@
+ using System.Collections.Generic;
+ using Terraria.GameContent.Events;
+@@ -7,19 +_,22 @@
+ 
+ namespace Terraria.GameContent.UI
  {
- 	public class EmoteBubble
+-	public class EmoteBubble
++	public partial class EmoteBubble
  	{
 -		private static int[] CountNPCs = new int[670];
+-		public static Dictionary<int, EmoteBubble> byID = new Dictionary<int, EmoteBubble>();
 +		internal static int[] CountNPCs = new int[NPCID.Count];
- 		public static Dictionary<int, EmoteBubble> byID = new Dictionary<int, EmoteBubble>();
++		internal static Dictionary<int, EmoteBubble> byID = new Dictionary<int, EmoteBubble>(); // TML: Changed from public to internal. Use EmoteBubbles instead for code readability
  		private static List<int> toClean = new List<int>();
- 		public static int NextID;
+-		public static int NextID;
+-		public int ID;
++		private static int NextID; // TML: Changed from public to private
++		internal int ID; // TML: Changed from public to internal
```

Comment:
No need for this access change

Reply:
This is for notifying modders to use `emote` field for real emote ID and avoid confusing. (The `ID` field here is like `whoAmI` in NPC)